apps/src/music/redux/musicRedux.ts

```diff
@@ -61,5 +113,13 @@ export const {
   setIsPlaying,
   setCurrentPlayheadPosition,
   selectBlockId,
-  clearSelectedBlockId
+  clearSelectedBlockId,
+  toggleTimelinePosition,
+  setShowInstructions,
+  setInstructionsPosition,
+  toggleInstructions,
+  advanceInstructionsPosition,
+  showBeatPad,
+  hideBeatPad,
+  toggleBeatPad
 } = musicSlice.actions;
```

Comment:
I wonder if we'll get to the point where it feels like there are too many things in this list? :)  Is there an idiomatic way to split these up?

Reply:
Good question :) I'm not sure if there is, besides splitting the music reducer further into sub-reducers. It seems like our current philosophy is to create one reducer (or in this case, slice) per general area, e.g. lab. I'm not sure if it would be best to split beyond that (maybe MusicUIState, MusicPlaybackState?). I also do have a preference to err on the side of having multiple actions with specific, narrow use cases to defer as much logic and control to the reducer - for example, `showBeatPad` and `hideBeatPad` could also just be a `setShowBeatPad(isShowing: boolean)`, but in our code, we are only ever explicitly hiding, showing, or toggling the beat pad, so I thought it would be clearer to expose specific actions for each, and not have the views need to know about how the `isShowing` payload should be structured (even if it's pretty self explanatory, i.e. false = hide beat pad). However, that does contribute to more action functions here.